docs/examples/SS-5--form.js

```diff
@@ -0,0 +1,280 @@
+import { currentOrPastDateConfig } from 'us-forms-system/lib/js/definitions/currentOrPastDate';
```

Comment:
This use of `currentOrPastDateConfig` uses the new version of this definition, so perhaps we want to be consistent with how it was used in the other file as well.